src/main.ts

```diff
@@ -1,3 +1,5 @@
+import '@angular/compiler';
```

Comment:
Why do we need to import the compiler into the app?

Reply:
Newer versions of Angular don't include the JIT compiler by default.

Reply:
Angular starter apps do not have this line included and have `@angular/compiler` as dependency, so I wonder if there is something in the app that was configured incorrectly in the past. Do you get an error without that line?

Reply:
Yeah, the app doesn't run otherwise. In the browser console:
> Error: The injectable 'InAppBrowser' needs to be compiled using the JIT compiler, but '@angular/compiler' is not available.
> JIT compilation is discouraged for production use-cases! Consider using AOT mode instead.
> Alternatively, the JIT compiler should be loaded by bootstrapping using '@angular/platform-browser-dynamic' or '@angular/platform-server',
> or manually provide the compiler with 'import "@angular/compiler";' before bootstrapping.

I looked into it a bit, and it sounds like JIT is better for dev, so I figured it'd be better to keep it as-is.

Reply:
Ah that's strange. Looks like we are loading the JIT compiler with `@angular/platform-browser-dynamic` but it doesn't seem to be working. Let's keep it as you had it for now.